src/DrupalArtifactBuilderGit.php

```diff
@@ -207,4 +207,30 @@ protected function assertArtifactExists() {
     }
   }
 
+  /**
+   * Get the name of the repository branch.
+   *
+   * @param \Symfony\Component\Console\Input\InputInterface
+   *   Shell input, used to get the branch from options.
+   *
+   * @return string
+   *   Branch name.
+   */
+  protected function getBranch(InputInterface $input) {
+    $branch_from_input = $input->getOption('branch');
```

Comment:
for completeness we may check if the provided branch is valid.

Reply:
Branches are generated dynamically for artifacts, so I think we can omit that.